modules/sdcore-control-plane-k8s/outputs.tf

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+output "amf_app_name" {
```

Comment:
What about integration endpoints?
```
output "fiveg_n2_endpoint" {
  description = "Name of the endpoint used to provide information on connectivity to the N2 plane."
  value       = "fiveg_n2_endpoint"
}
And so on...

Reply:
The required integration points are added.